web/client/utils/DOMUtil.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+/**
+ * Collection of tool to work on DOM
```

Comment:
if possible for new files, try to use es6 import export syntax
so no need of DOMUtil object

```
export const scrollIntoViewId = (viewId) => {
        const node = document.getElementById(viewId);
        if (node && node.scrollIntoView) {
            node.scrollIntoView({ behavior: 'smooth', block: 'start' });
        }
    }
};
```